autoload/importjs.vim

```diff
@@ -52,18 +65,25 @@ function importjs#ExecCommand(command, arg)
     return
   endtry
 
-  let result = json_decode(resultString)
+  if (resultString != "")
+    return importjs#ParseResult(resultString, a:command)
+  endif
+endfunction
+
+function importjs#ParseResult(resultString, command)
+  let result = json_decode(a:resultString)
 
   if (has_key(result, 'error'))
     echoerr result.error
     return
   endif
 
-  if (a:command == "goto" && has_key(result, 'goto'))
+  if ((a:command == "goto" || a:command == "") && has_key(result, 'goto'))
```

Comment:
Not sure what the reason for comparing the command to command from result, but there didn't seem to be a good way to do this since neovim jobs are async